eng/ci_tools.txt

```diff
@@ -13,6 +13,7 @@ wrapt==1.12.1; python_version <= '3.10'
 wrapt==1.15.0; python_version >= '3.11'
 typing-extensions<=4.6.3
 pyproject-api<1.6
+cibuildwheel==2.16.2 ; python_version > '3.8'
```

Comment:
Does this mean we can't use cibuildwheel to produce artifacts for older Python versions? Or just that our build machines will use > 3.9? I assume the latter?

Reply:
`cibuildwheel` refuses to install on python < 3.8. Just not even supported anymore. `eng/ci_tools.txt` is installed as a prestep, but we don't actually need `cibuildwheel` during testing, just on the `build` step. That's why I can safely add this filter.

Reply:
`eng/ci_tools.txt` = pinned stuff that are used to run ci checks. `tox` version is here. `coverage`. Etc. The "surface" tools.
`eng/test_tools.txt` = pinned reqs if you need to run tests.